nilearn/regions/region_extractor.py

```diff
@@ -289,12 +289,11 @@ class RegionExtractor(NiftiMapsMasker):
         parameter indicates that we keep only those voxels which have
         intensities more than this value.
         Default='ratio_n_voxels'.
-
     %(extractor)s
-    smoothing_fwhm : :obj:`float`, optional
-        To smooth an image to extract most sparser regions. This parameter
-        is passed to `connected_regions` and exists only for extractor
-        'local_regions'. Please set this parameter according to maps
+    %(smoothing_fwhm)s
+        Use this parameter to smooth an image to extract most sparser regions.
+        This parameter is passed to `connected_regions` and exists only for
+        extractor 'local_regions'. Please set this parameter according to maps
```

Comment:
I don' understand this sentence

Reply:
I tried to clarify it in 4a2c0fa. Basically, `smoothing_fwhm` will only be considered if extraction option is set to `local_regions`.